docs/helm/templates/restricted-admin.yaml

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
This file doesn't grant admin permissions, so having the admin in the file name is misleading. Suggest using robot-deployer instead.

Reply:
Done